docs/version0.30.md

```diff
@@ -0,0 +1,50 @@
+<!--
+* Copyright (c) 2021, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.30.0
+
+The following new features and notable changes since v 0.29.0 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- [Changes to the shared classes cache generation number](#changes-to-the-shared-classes-cache-generation-number)
+
+## Features and changes
+
+### Binaries and supported environments
+
+OpenJ9 release 0.30.0 supports OpenJDK 8, 11 and 17.
+
+AArch64 Linux is now a fully supported, production-ready target for OpenJDK 8 and 11.
```

Comment:
We don't need this line, as AArch64 was already introduced in 0.29

Reply:
Removed.